src/backend/DonateTests/Services/GetWarehousesTest/GetWarehousesActionTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Core.Exceptions;
+using Autofac;
+using Core.Database;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using TestsBase;
+using Donate.Actions.DonateForm.GetWarehouses;
+using Core.Database.Enums;
+using Core.Database.Models.Auth;
+using Core.Setup.Auth0;
+using Microsoft.AspNetCore.Mvc;
+using Core.Database.Models.Core;
+using System.Diagnostics.Metrics;
+using System.Net;
+using Donate;
+using Core;
+using Core.Setup.Enums;
+
+namespace DonateTests.Services.GetWarehousesTest
+{
+    [TestClass()]
+    public class GetWarehousesActionTests
+    {
+        private readonly List<Module> _usedModules = new()
+        {
+            new CoreModule(WebApiUserProviderType.None),
+            new DonateModule(),
+        };
+
+        [TestMethod()]
+        public async Task GetWarehousesActionTest_GetWarehouses_ReturnsWareHouses()
+        {
+            using var toolkit = new TestsToolkit(_usedModules);
+            var context = toolkit.Resolve<AppDbContext>();
+            var action = toolkit.Resolve<GetWarehousesAction>();
+
+            // Act
+            var result = await action.Execute();
+
+            // Assert
+            Assert.IsNotNull(result);
```

Comment:
To chyba nie jest zbyt dobry pomysl, bo to nigdy nie będzie nullem. Zawsze będzie `OkObjectResult`.
Sprawdź czy wynikowa kolekcja nie jest pusta.

Reply:
Masz rację, poprawione.